### PR TITLE
Prevent podman-env command on Windows

### PR DESCRIPTION
**Pull request by gbraad**
Follow-up to #1001

### CODE REVIEW

**Comment by zeenix** [comment]
LGTM

File: pkg/crc/preflight/preflight_checks_common.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/code-ready/crc/pkg/crc/oc"
 	"github.com/code-ready/crc/pkg/crc/podman"
 	"github.com/code-ready/crc/pkg/embed"
+	crcos "github.com/code-ready/crc/pkg/crc/os"
```
Comment (praveenkumar) [inline pkg/crc/preflight/preflight_checks_common.go]: @gbraad this should be `crcos "github.com/code-ready/crc/pkg/os"` you added `crc` in between and it failed to buld :)

Comment (praveenkumar) [reply pkg/crc/preflight/preflight_checks_common.go]:
```patch
diff --git a/pkg/crc/preflight/preflight_checks_common.go b/pkg/crc/preflight/preflight_checks_common.go
index f485845..1dbf60d 100644
--- a/pkg/crc/preflight/preflight_checks_common.go
+++ b/pkg/crc/preflight/preflight_checks_common.go
@@ -11,7 +11,7 @@ import (
        "github.com/code-ready/crc/pkg/crc/oc"
        "github.com/code-ready/crc/pkg/crc/podman"
        "github.com/code-ready/crc/pkg/embed"
-       crcos "github.com/code-ready/crc/pkg/crc/os"
+       crcos "github.com/code-ready/crc/pkg/os"
 )
 
 var genericPreflightChecks = [...]PreflightCheck{
```

**Comment by praveenkumar** [comment]
Force pushed, closing it.

**Pull request closed without merging**